workspaces/util/src/__tests__/timestampToYearMonthDay-test.ts

```diff
@@ -0,0 +1,28 @@
+import { describe, expect, it } from "@jest/globals";
+
+import { timestampToYearMonthDay } from "../timestampToYearMonthDay";
+
+describe("timestampToYearMonthDay", () => {
+  it("returns formated timestamp correctly with default separator", () => {
```

Comment:
I believe we missed one last typo 😃 

```suggestion
  it("returns formatted timestamp with default separator", () => {
```